deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -478,17 +476,6 @@ def merge_input_provider(index, item):
         mat_results, telemetry_time_hb + telemetry_time_merge
     )
 
-    cluster_util: ClusterUtilizationOverTimeRange = kwargs.get("cluster_util")
-
-    if cluster_util:
-        compaction_audit.set_total_cpu_seconds(cluster_util.total_vcpu_seconds)
-        compaction_audit.set_used_cpu_seconds(cluster_util.used_vcpu_seconds)
-        compaction_audit.set_used_memory_gb_seconds(cluster_util.used_memory_gb_seconds)
-        compaction_audit.set_total_memory_gb_seconds(
-            cluster_util.total_memory_gb_seconds
-        )
-        compaction_audit.set_cluster_cpu_max(cluster_util.max_cpu)
```

Comment:
Remove the keys from `compaction_session_audit_info`. Or document on them saying they are not current emitted until we figure out how to measure vCPUSeconds at partition level.